packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -24,6 +24,13 @@ const {
 const { pickSelectionParams } = require('./params');
 const { applyTransforms } = require('./attributes');
 
+const transformLoadParamsToQuery = (uid, field, params = {}, pagination = {}) => {
+  return {
+    ...transformParamsToQuery(uid, { populate: { [field]: params } }).populate[field],
+    ...pagination,
+  };
+};
+
```

Comment:
I did this function because `transformParamsToQuery` is not adapted for the load function. The load function takes subPopulates params which is a bit different than regular params (for instance there cannot be page/pageSize in subPopulates but they can exist in regular params).
We can make a call to talk about it

Reply:
I do not quite understand this part.
I understand in subPopulates there can not be any page/pageSize but not how transformPAramsToQuery was not handling that before.
Could we have that meeting to explain this? 🙏🏻